cmd/podman/create.go

```diff
@@ -117,6 +140,7 @@ func createCmd(c *cli.Context) error {
 	options = append(options, libpod.WithShmSize(createConfig.Resources.ShmSize))
 	options = append(options, libpod.WithGroups(createConfig.GroupAdd))
 	options = append(options, libpod.WithIDMappings(*createConfig.IDMappings))
+	options = append(options, libpod.WithRootfs(createConfig.Rootfs))
```

Comment:
`fs` should be capitalized (throughout), for consistency with `libpod.WithRootFSFromImage`.